src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -38,6 +41,12 @@ public AbstractSqlDialect(final ConnectionFactory connectionFactory, final Adapt
         this.properties = properties;
     }
 
+    protected static List<String> addAdditionalSupportedProperties(final List<String> additionalPropertiesToSupport) {
```

Comment:
The method does not do what the name says. It has a side effect.

Reply:
fixed